### PR TITLE
fix: return "unknown command" instead of "no help topic" error when unknown (sub)command is passed

### PR DESCRIPTION
**Pull request by danxmoran**
Follow-up to #312

### CODE REVIEW

File: cmd/influx/bucket_schema.go [hunk 1]
```diff
@@ -36,8 +36,9 @@ func getBucketSchemaClient(ctx *cli.Context) bucket_schema.Client {
 
 func newBucketSchemaCmd() cli.Command {
 	return cli.Command{
-		Name:  "bucket-schema",
-		Usage: "Bucket schema management commands",
+		Name:   "bucket-schema",
+		Usage:  "Bucket schema management commands",
+		Before: middleware.NoArgs,
```
Comment (lesam) [inline cmd/influx/bucket_schema.go:41]: NoArgs is a weird name for this - I would call it something like `SubcommandValid`

**Review by lesam — APPROVED** [review]
One nitpick about existing naming

File: cmd/influx/bucket_schema.go (hunk 1 above)
Comment (danxmoran) [reply cmd/influx/bucket_schema.go:41]: The name & error message mirror the `cobra` equivalent. It's meant to say "this command doesn't take any positional args, so if you see one try parsing it as a subcommand" which then leads to "unknown command" errors... agreed it's not always the most ideal way to think about it.

**Comment by slai** [comment]
@danxmoran LGTM, thanks for taking this over and merging it

**Pull request merged**